### ISSUE TITLE
Can't get Hyper to use my font

### ISSUE THREAD

**Issue by deiga** [zeit/hyper#1988]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS Sierra 10.12.5
- **Hyper.app version**: 1.3.3.1754
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: [.hyper.js](https://gist.github.com/b4a9d48a9c7d3ef044f35502177dfc3b)
- **Relevant information from devtools** 
```
config.getConfig().fontFamily
""Meslo LG M Regular for Powerline Nerd Font Complete","Meslo LG S DZ Regular for Powerline Nerd Font Complete","Meslo LG S Regular for Powerline Nerd Font Complete", "Meslo LG L Regular for Powerline Nerd Font Complete", Menlo, "DejaVu Sans Mono", Consolas, "Lucida Console", monospace"
```  
- **The issue is reproducible in vanilla Hyper.app**: yes 

## Issue

I've set my custom fontFamily in the configuration and restarted Hyper but for some reason the font doesn't come into effect. 
My fonts are installed in `~/Library/Fonts`.
My locale is set to UTF-8: 
```
$ locale
LANG="en_GB.UTF-8"
LC_COLLATE="en_GB.UTF-8"
LC_CTYPE="en_GB.UTF-8"
LC_MESSAGES="en_GB.UTF-8"
LC_MONETARY="en_GB.UTF-8"
LC_NUMERIC="en_GB.UTF-8"
LC_TIME="en_GB.UTF-8"
LC_ALL="en_GB.UTF-8"
```

**Comment by deiga** [comment]
Apparently installing Nerd Fonts with brew messes the font names up.
`"MesloLGL Nerd Font"` is the correct font name for Meslo Nerd Fonts

**Comment by jackmoody11** [comment]
Just for anybody else who is wondering, for Fira Code, it is `"FiraCode Nerd Font"`.

**Comment by sbanders1** [comment]
For others who might be wrestling with this, I also was having issues where the font would create tiny boxes with question marks. Turns out it was because my version of Windows 10 Pro was not activated. I found this out by trying to search for Fonts in the settings, and was unable to search for any of the Nerd Fonts I thought I had downloaded. Activating Windows 10 Pro fixed the problem as soon as I reloaded.
